assets/stylesheets/shared/_forms.scss

```diff
@@ -159,19 +159,23 @@ input[type=radio] + span {
 }
 
 input[type=checkbox] {
-	&:checked:before {
-		@include clear-text;
+	border-radius: 2px;
 
-		content: '\f418';
-		margin: -4px 0 0 -5px;
-		float: left;
+	&:checked {
+		border-color: $blue-medium;
+		background-color: $blue-medium;
+	}
+
+	&:checked:before {
+		content: url( "/calypso/images/checkbox-icons/checkmark.svg" );
```

Comment:
Is there a better way to do this without injecting the SVG in with CSS? Is there something stopping us from hiding/rendering the checkbox in the jsx?

Reply:
My biggest concern is that we're duplicating a Gridicon in the `/images/` folder that will need to be manually updated if Gridicon's changes.